From e537d62526037fee73757b5bd44bcf23df101ada Mon Sep 17 00:00:00 2001
From: Peter Yin <peter.yin@quantatw.com>
Date: Mon, 21 Jul 2025 13:14:41 +0800
Subject: [PATCH] ARM: dts: aspeed: Harma: Enable i2c slave timeout

Enable the I2C slave timeout property in the Harma device tree to
prevent the slave interface from hanging due to incomplete or abnormal
I2C transactions, such as those occurring during MMC reboot.
---
 arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts
index a294f2b7925b..c52ba08b0e69 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts
@@ -468,6 +468,7 @@ &i2c9 {
 
 	mctp-controller;
 	multi-master;
+	i2c-scl-clk-low-timeout-us = <31744>;
 
 	mctp@10 {
 		compatible = "mctp-i2c-controller";
-- 
2.53.0-Meta

